### DISCUSSION TITLE
Where is the Callback Documentation?

### DISCUSSION THREAD

**Discussion by alvingalangcitaka** [jbergler/hass-ttlock#157] [General]
I tried your integration and it seems to work for lock and unlock, but I also need another sensor that is installed on the door. So I create my own template to actually do the same thing like this integration.

I'm wondering how did you get the list of event description like [here](https://github.com/jbergler/hass-ttlock/blob/351e32ab526739dd114c386447d4ecfc64b9e063/custom_components/ttlock/models.py#L161)?
I can't find them on ttlock open api documentation.

**Comment by square-spade** [comment]
Its [here](https://euopen.ttlock.com/document/doc?urlName=cloud%2FlockRecord%2FrecordTypeFromCloudEn.html)

(and the recordNotify base info is [here](https://euopen.ttlock.com/document/doc?urlName=cloud%2FlockRecord%2FnotifyEn.html))

**Comment by alvingalangcitaka** [comment]
thank you
